azure-pipelines.yml

```diff
@@ -77,6 +77,14 @@ steps:
     cargo test --verbose --all --features "test unstable"
   displayName: test
 
+- script: |
```

Comment:
```suggestion
# TODO: Use the latest version of tarpaulin once our code compiles with it
- script: |
```

Let's add a comment here to help us remember why we're using an old version of tarpaulin.